train.py

```diff
@@ -139,6 +140,8 @@ def main(job_config: JobConfig):
 
     # loss function to be shared by Pipeline Parallel and SPMD training
     def loss_fn(pred, labels):
+        if isinstance(pred, DTensor):
+            pred._local_tensor = pred._local_tensor.contiguous()
```

Comment:
please comment that this is a temporary fix to enable async TP for full model compile